fe49-onl-maryia-dauhan/src/features/tab/tabs.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  id: number;
+  tabTitle: string;
+  title: string;
+  content: string;
+  disabled?: boolean;
+}
+
+const tabs: Props[] = [
+  {
+    id: 0,
+    tabTitle: 'Tab 1',
+    title: 'Title 1',
+    content:
+      'Душа моя озарена неземной радостью, как эти чудесные весенние утра',
+  },
+  {
+    id: 1,
+    tabTitle: 'Tab 2',
+    title: 'Title 2',
+    content:
+      'Проснувшись однажды утром после беспокойного сна, Грегор Замза обнаружил, что он у себя в постели превратился в страшное насекомое.',
+  },
+  {
+    id: 2,
+    tabTitle: 'Tab 3',
+    title: 'Title 3',
+    content:
+      'Далеко-далеко за словесными горами в стране гласных и согласных живут рыбные тексты. Вдали от всех живут они в буквенных домах на берегу Семантика большого языкового океана. ',
+  },
+  {
+    id: 3,
+    tabTitle: 'Tab 4',
+    title: 'Title 4',
+    content:
+      'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.',
+    disabled: true,
+  },
+];
+
+export const TabDefault = () => {
+  const [currentTab, setSurrentTab] = useState(0);
+
+  const handleTabClick = (id: number) => {
+    setSurrentTab(id);
+  };
+
+  return (
+    <TabWrapper>
+      <TabList>
+        {tabs.map((tab, id) => (
+          <TabItem>
+            <TabButton
+              type='button'
+              key={id}
```

Comment:
key should be placed on the first root element inside callback (`<TabItem />`)

Reply:
fixed. thank you